scripts/hid_configurator/modules/dfu.py

```diff
@@ -347,10 +373,11 @@ def __del__(self):
 
 
 def fwinfo(dev):
-    success, fetched_data = dev.config_get('dfu', 'fwinfo')
+    success_data, fetched_data = dev.config_get('dfu', 'fwinfo')
+    success_bootloader, fetched_bootloader_variant = dev.config_get('dfu', 'bootloader_var')
 
-    if success and fetched_data:
-        fw_info = FwInfo(fetched_data)
+    if success_data and success_bootloader and fetched_data and fetched_bootloader_variant:
```

Comment:
btw. From this point, the Python scripts related to the DFU would not longer be compatible with previous firmware. Maybe it's worth to inform about it in docs or release notes ?

Reply:
Ok, will put information in release note.